recipe/cmake.patch

```diff
@@ -0,0 +1,15 @@
+--- /dev/null	2019-01-16 21:39:07.828957088 +0100
++++ CMakeLists.txt	2019-01-17 03:36:35.265918586 +0100
+@@ -0,0 +1,12 @@
++cmake_minimum_required(VERSION 3.6)
++project(NTL)
++
++set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
++
++file(GLOB SOURCES src/*.cpp)
++include_directories(include)
++add_library(${PROJECT_NAME} SHARED ${SOURCES})
++
++install(TARGETS ${PROJECT_NAME} DESTINATION lib/${PROJECT_NAME})
```

Comment:
This needs to be changed. See https://github.com/conda-forge/planarity-feedstock/pull/11/files#diff-81f7f2242cc2104fe1f0862599e63025R10